src/components/o-accordion/index.scss

```diff
@@ -1,3 +1,3 @@
-.m-accordion {
+.o-accordion {
```

Comment:
And the file which uses this class?

Reply:
does not exists

Reply:
i remove it

Reply:
wait, but it should. Thought custom element are by default `inline` and in most cases we want them to be `display: block`.

Reply:
ah yes true sorry i revert :)

Reply:
no hold on the aricle underneath has display block as default so is it not enough that?